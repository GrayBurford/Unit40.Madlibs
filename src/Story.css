
.Story {
    font-size: 2rem;
}