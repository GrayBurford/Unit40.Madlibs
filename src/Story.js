import React from 'react';
import './Story.css';

// Basic component to render a new Madlib generated by MadlibsForm
function Story ({ noun1, noun2, adjective, color }) {
    
    return (
        <div className='Story'>
            <h4>{`There once was a ${color} ${noun1} who loved a ${adjective} ${noun2}`}</h4>
        </div>
    )
}


export default Story;