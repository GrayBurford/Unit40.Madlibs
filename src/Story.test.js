import React from "react";
import { render } from "@testing-library/react";
import Story from "./Story";

// smoke test
test('Renders without crashing', () => {
    render(<Story />);
})


// snapshot test
test('Matches snapshot', () => {
    const { asFragment } = render(<Story />);
    expect(asFragment()).toMatchSnapshot();
})