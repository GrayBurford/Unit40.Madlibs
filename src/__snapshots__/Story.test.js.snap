// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Story"
  >
    <h4>
      There once was a undefined undefined who loved a undefined undefined
    </h4>
  </div>
</DocumentFragment>
`;
